src/web/app/pages-instructor/instructor-search-page/instructor-search-bar/instructor-search-bar.component.html

```diff
@@ -9,11 +9,21 @@
     </div>
     <div class="form-group">
       <div class="input-group">
-        <input type="text" class="form-control" [(ngModel)]="searchKey" (keyup.enter)="search()"
+        <input type="text" class="form-control" [(ngModel)]="searchParams.searchKey" (keyup.enter)="search()"
             placeholder="Your search keyword">
           <div class="input-group-append">
             <button class="btn btn-primary" (click)="search()">Search</button>
           </div>
+        <div class="input-group mt-md-1">
+          <div class="col-md-3">
+            <input type="checkbox" class="form-check-inline" id="studentsCheckbox" [(ngModel)]="searchParams.isSearchForStudents">
```

Comment:
Why you current implementation work is because here you do 2-way binding for  `searchParams.isSearchForStudents` (i.e. you are still changes the value in the child component)

Reply:
Ah i see. Okay, will change the implementation